pkg/cloud/services/compute/bastion.go

```diff
@@ -78,8 +79,8 @@ func (s *Service) CreateBastion(openStackCluster *infrav1.OpenStackCluster, clus
 
 	out, err := createInstance(s, clusterName, input)
 	if err != nil {
-		record.Warnf(openStackCluster, "FailedCreateServer", "Failed to create bastion: %v", err)
-		return nil, fmt.Errorf("create new server err: %v", err)
+		record.Warnf(openStackCluster, "FailedCreateServer", "Failed to create server %s: %v", name, err)
+		return nil, err
 	}
```

Comment:
nit:

can you move the empty line one line up?
So like this
```go
if err != nil {
   record.Warn...
   return nil, err
}
<empty line>
record.Eventf...
return out, nil
```
(everywhere where we are using this pattern)

Reply:
Updated.